packages/palette/src/elements/Checkbox/Checkbox.story.tsx

```diff
@@ -0,0 +1,38 @@
+import { action } from "@storybook/addon-actions"
+import React, { useState } from "react"
+import { Text } from "../Text"
+import { Checkbox } from "./Checkbox"
+
+export default {
+  title: "Components/Checkbox",
+}
+
+export const States = () => {
+  const states = [{}, { selected: true }, { disabled: true }, { error: true }]
```

Comment:
I'll probably add my thing for this https://github.com/dzucconi/storybook-states and flesh out the stories with it. Very useful for this sort of thing.